inputfiles/sample.webidl.json

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
Why do we need this file?

Reply:
This is for F# type provider to compute JSON types. Otherwise it has to read the whole file which will take more time.

Reply:
thanks. we should name it as such then.

Reply:
The current sample.xml has the same purpose, do we also want to rename it? typeprovider.xml/json?

https://github.com/Microsoft/TSJS-lib-generator/blob/048c8bd0c2d47b5c234f47a4490b2e236d11688b/TS.fsx#L59